<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewLens</class>
 <widget class="QDialog" name="NewLens">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>286</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>385</width>
    <height>286</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>385</width>
    <height>286</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AvisionR - New Lens</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:rgb(255, 255, 255);
background-color: #262626;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Type:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Type">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #686868;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Add-Ons:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_AddOns">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #686868;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Price: $</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_Price_2">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>20</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #686868;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_12">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Units:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_units_2"/>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QCommandLinkButton" name="commandLinkButton_add_2">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="commandLinkButton_cancel_2">
       <property name="maximumSize">
        <size>
         <width>179</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
